src/mame/ibm/thinkpad600.cpp

```diff
@@ -217,6 +303,31 @@ void thinkpad600_state::thinkpad600(machine_config &config)
 }
 
 
+ROM_START(thinkpad760xd)
+	ROM_REGION( 0x80000, "bios", 0 )
+	ROM_LOAD( "e28f004_89g8164_rev37_h1897m.u17", 0x00000, 0x80000, CRC(6092594f) SHA1(25681e4952a432e1170f69ae75f3260245b6b44b) ) // BIOS
+
+	ROM_REGION( 0x0f780, "mcu", 0)
+	ROM_LOAD( "ibm_hd6433436a18f_40h8792.u39",    0x00000, 0x0f780, NO_DUMP ) // Mask ROM, undumped
+
+	ROM_REGION( 0x00080, "seeprom", 0)
+	ROM_LOAD( "st93c46c.u30",                     0x00000, 0x00080, CRC(22cac7b5) SHA1(ee48ecf5d59e243e9afb0ca7e41ed8437eec8097) ) // BIOS settings
+ROM_END
+
+ROM_START(thinkpad600)
+	ROM_REGION( 0x80000,  "bios", 0 )
+	ROM_LOAD( "tms28f004b_18l9949_rev16-i2298m.u76",  0x00000, 0x80000, CRC(00a52b32) SHA1(08db425b8edb3a036f22beb588caa6f050fc8eb2) )
+
+	ROM_REGION(0x0f780, "mcu", 0)
+	ROM_LOAD( "hd64f3437tf_10l9950_rev08_i2798m.u32", 0x00000, 0x0f780, CRC(546ec51c) SHA1(5d9b4be590307c4059ff11c434d0901819427649) )
+
+	ROM_REGION(0x00080, "seeprom", 0)
+	ROM_LOAD( "atmel_24c01a.u49",                     0x00000, 0x00080, CRC(9a2e2a18) SHA1(29e2832c97bc93debb4fb09fcbed582335b57efe) ) // BIOS settings
+
+	ROM_REGION(0x00c39, "plds", 0)
+	ROM_LOAD( "atf1500al-modemboard.u12",             0x00000, 0x00c39, CRC(7ecd4b79) SHA1(b69ef5fe227b466f331f863ba20efd7e23056809) ) // On modem PCB
+ROM_END
```

Comment:
This will introduce a regression. 
The bios area maps to `pci:07.0`, using `bios` without a correlated `pci:07.0` will throw a segmentation fault when attempted to be run. So either change these `bios` rom regions to `pci:07.0` for all the sets or, if you absolutely need the `bios` rom region, do something of the like:

```c++
ROM_START(thinkpadXXXyyy)
	ROM_REGION( 0x80000, "bios", 0 )
	ROM_LOAD( // fill in details

	ROM_REGION( 0x80000, "pci:07.0", ROMREGION_ERASEFF )
    // TODO: unverified
	ROM_COPY( "bios", 0x00000, 0x00000, 0x80000 )
```

Reply:
Done, thanks!